sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -20,17 +21,39 @@ public static async Task Main(string[] args)
         {
             Options options = null;
             ParserResult<Options> result = Parser.Default.ParseArguments<Options>(args)
-                .WithParsed(parsedOptions => { options = parsedOptions; })
-                .WithNotParsed(errors => Options.HandleParseError(errors));
+                .WithParsed(parsedOptions =>
+                    {
+                        options = parsedOptions;
+                    })
+                .WithNotParsed(errors =>
+                    {
+                        Environment.Exit(1);
+                    });
+
+            if (options.GetLoginMethod() == LoginMethod.AppId
```

Comment:
Special handling for optionally required parameter.

Reply:
you can use mutually exclusive sets for your options so this case automatically gets handled for you.

Reply:
Got an example? I tried but couldn't find one.